mmv1/third_party/terraform/services/alloydb/resource_alloydb_cluster_test.go

```diff
@@ -1167,3 +1168,136 @@ resource "google_compute_global_address" "private_ip_alloc" {
   
 `, context)
 }
+
+func TestAccAlloydbCluster_withMaintenanceWindows(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckAlloydbClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAlloydbCluster_withMaintenanceWindows(context),
+			},
+			{
+				ResourceName:            "google_alloydb_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"initial_user", "cluster_id", "location"},
+			},
+		},
+	})
+}
+
+func testAccAlloydbCluster_withMaintenanceWindows(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_alloydb_cluster" "default" {
+  provider   = google-beta
```

Comment:
Is the maintenance windows field beta-only? If so, it should be marked min_version: beta in the YAML file. Otherwise, remove this provider line

Reply:
Note from office hours.
On launch this will be available both the ga and beta apis.

Reply:
Should it be available in the GA provider then? Or will it still be considered "beta"?

Reply:
Correct. It should be in both providers. Not considered beta.